Dockerfile

```diff
@@ -12,13 +12,57 @@ LABEL changelog-url "https://github.com/CommunityHoneyNetwork/communityhoneynetw
 VOLUME /tls
 
 ENV playbook "chnserver.yml"
+ENV TZ "America/New_York"
```

Comment:
The non-interactive seems like a good item for all our container builds, but the TZ seems presumptuous.